packages/bun-usockets/src/loop.c

```diff
@@ -397,6 +397,36 @@ void us_internal_dispatch_ready_poll(struct us_poll_t *p, int error, int events)
             }
             break;
         }
+        case POLL_TYPE_UDP: {
+            struct us_udp_socket_t *u = (struct us_udp_socket_t *) p;
+            // do cleanup
+            if (u->closed) {
```

Comment:
this in theory could lose readable data (see above, there was a similar bug before)